tests/utils.ts

```diff
@@ -22,6 +22,15 @@ export function generateVarName(prefix = 'v'): string {
   return `${prefix}_${id}`;
 }
 
+/**
+ * Log browser type and version.
+ * @param browser
+ */
+export function logBrowserInfo(browser: Browser): void {
+  // eslint-disable-next-line no-console
+  console.log('Browser:', browser.browserType().name(), browser.version());
+}
```

Comment:
Honestly, we should probably just log the browser version, app version and other information when the app starts up. Would be useful for the support logs as well.

Reply:
Updated